<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_lts">
       <attribute name="title">
        <string>LTS</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget_LTS"/>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_LTS">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_stable">
       <attribute name="title">
        <string>Stable</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_Stable"/>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget_Stable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_daily">
       <attribute name="title">
        <string>Daily</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_Daily"/>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget_Daily"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLocal_path"/>
    <addaction name="actionClear_Local_Downloads"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionLocal_path">
   <property name="text">
    <string>Set Download Path</string>
   </property>
  </action>
  <action name="actionClear_Local_Downloads">
   <property name="text">
    <string>Clear Local Downloads</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
